tests/sql/operators/test_merge.py

```diff
@@ -109,7 +109,10 @@ def run_merge(target_table: Table, source_table: Table, merge_parameters, mode):
 )
 @pytest.mark.parametrize(
     "database_table_fixture",
-    [{"database": Database.SNOWFLAKE}, {"database": Database.BIGQUERY}],
+    [
```

Comment:
@pankajkoti  I think this is a formatting change, it should be part of a different PR, right?

Reply:
I believe formatting change can go in this PR.